### PR TITLE
Release configuration on-boarding example

### PR DESCRIPTION
**Pull request by mgreenegit**
Republishing previous work as new PR.

### CODE REVIEW

File: 101-automation-configuration/nested/provisionDNS.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        "Description": "The DNS address(es) of the DNS Server(s) used by the virtual network"
+      }
+    }
+  },
+  "resources": [
+    {
+      "name": "[parameters('virtualNetworkName')]",
+      "type": "Microsoft.Network/virtualNetworks",
+      "location": "[parameters('location')]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-automation-configuration/nested/provisionDNS.json]: update apiVersion

File: 101-automation-configuration/nested/provisionNetwork.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "publicIPAddressType": {
+      "type": "string"
+    }
+  },
+  "resources": [
+    {
+      "name": "[parameters('virtualNetworkName')]",
+      "type": "Microsoft.Network/virtualNetworks",
+      "location": "[parameters('location')]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-automation-configuration/nested/provisionNetwork.json]: update apiVersion

File: 101-automation-configuration/nested/provisionNetwork.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        "addressSpace": {
+          "addressPrefixes": [
+            "[parameters('virtualNetworkAddressRange')]"
+          ]
+        },
+        "subnets": "[parameters('virtualNetworkSubnets')]"
+      }
+    },
+    {
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-automation-configuration/nested/provisionNetwork.json]: update apiVersion

File: 101-automation-configuration/nested/provisionNetwork.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      ],
+      "properties": {
+        "publicIPAllocationMethod": "[parameters('publicIPAddressType')]"
+      }
+    },
+    {
+      "name": "[parameters('nicName')]",
+      "type": "Microsoft.Network/networkInterfaces",
+      "location": "[parameters('location')]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-automation-configuration/nested/provisionNetwork.json]: update apiVersion

File: 101-automation-configuration/nested/provisionServer.json [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+    "nodeConfigurationName": {
+      "type": "string",
+      "defaultValue": "domainControllerConfig.localhost"
+    }
+  },
+  "variables": {
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-10-31",
```
Comment (bmoore-msft) [inline 101-automation-configuration/nested/provisionServer.json]: update apiVersion

File: 101-automation-configuration/nested/provisionStorage.json [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      "apiVersion": "2015-06-15",
+      "location": "[parameters('location')]",
+      "properties": {
+        "accountType": "[parameters('storageAccountType')]"
+      }
+    },
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[concat(parameters('storageAccountNamePrefix'),'2')]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-automation-configuration/nested/provisionStorage.json]: update apiVersion

File: 101-automation-configuration/README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+# Azure Automation Managed Node template
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmgreenegit%2Fazure-quickstart-templates%2F101-automation-configuration%2F101-automation-configuration%2Fazuredeploy.json" target="_blank">
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/README.md]: fixed

File: 101-automation-configuration/azuredeploy.json [hunk 8]
```diff
@@ -0,0 +1,318 @@
...
+    "compileName": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique value to identify compilation job"
+      },
+      "defaultValue": "[guid(resourceGroup().id, deployment().name)]"
+    }
+  },
+  "variables": {
+    "automationAccountName": "[concat(parameters('prefix'), 'configuration')]",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: good catch

File: 101-automation-configuration/azuredeploy.json [hunk 9]
```diff
@@ -0,0 +1,318 @@
...
+          "addressPrefix": "[parameters('virtualNetworkSubnet')]"
+        }
+      }
+    ],
+    "VMName": "server",
+    "DataDisk": "DataDisk",
+    "DataDiskSize": 1000,
+    "nicName": "nic",
+    "virtual networkID": "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetworkName'))]",
+    "nicSubnetRef": "[concat(variables('virtual networkID'),'/subnets/',variables('virtualNetworkSubnetName'))]",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed

File: 101-automation-configuration/azuredeploy.json [hunk 10]
```diff
@@ -0,0 +1,318 @@
...
+    "provisionNetworkURL": "[uri(parameters('nestedTemplateBaseURL'), 'provisionNetwork.json')]",
+    "provisionServerUrl": "[uri(parameters('nestedTemplateBaseURL'), 'provisionServer.json')]",
+    "provisionDNSUrl": "[uri(parameters('nestedTemplateBaseURL'), 'provisionDNS.json')]",
+    "storageAccountNamePrefix": "[take(toLower(concat(parameters('prefix'), uniqueString(resourceGroup().id))),23)]"
+  },
+  "resources": [
+    {
+      "name": "provisionStorage",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2015-01-01",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed throughout

File: 101-automation-configuration/azuredeploy.json [hunk 11]
```diff
@@ -0,0 +1,318 @@
...
+          "storageAccountNamePrefix": {
+            "value": "[variables('storageAccountNamePrefix')]"
+          }
+        }
+      }
+    },
+    {
+      "name": "provisionConfiguration",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2016-02-01",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed throughout

File: 101-automation-configuration/azuredeploy.json [hunk 12]
```diff
@@ -0,0 +1,318 @@
...
+          "compileName": {
+            "value": "[parameters('compileName')]"            
+          }
+        }
+      }
+    },
+    {
+      "name": "provisionNetwork",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2015-01-01",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed throughout

File: 101-automation-configuration/azuredeploy.json [hunk 13]
```diff
@@ -0,0 +1,318 @@
...
+          "publicIPAddressType": {
+            "value": "[variables('publicIPAddressType')]"
+          }
+        }
+      }
+    },
+    {
+      "name": "provisionServer",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2015-01-01",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed throughout

File: 101-automation-configuration/azuredeploy.json [hunk 14]
```diff
@@ -0,0 +1,318 @@
...
+          "automationAccountName": {
+            "value": "[variables('automationAccountName')]"
+          }
+        }
+      }
+    },
+    {
+      "name": "provisionDNS",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2015-01-01",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed throughout

File: 101-automation-configuration/azuredeploy.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: good catch

File: 101-automation-configuration/azuredeploy.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: improved

File: 101-automation-configuration/azuredeploy.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/azuredeploy.json]: fixed throughout

File: 101-automation-configuration/nested/provisionServer.json [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces',parameters('NicName'))]"
+            }
+          ]
+        },
+        "diagnosticsProfile": {
+          "bootDiagnostics": {
+            "enabled": "true",
+            "storageUri": "[concat('http://',parameters('storageAccountNamePrefix'),parameters('diagnosticsStorageUriSuffix'))]"
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/nested/provisionServer.json]: fixed

File: 101-automation-configuration/nested/provisionStorage.json [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+        "Description": "The type of the Storage Account created"
+      },
+      "defaultValue": "Premium_LRS"
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[concat(parameters('storageAccountNamePrefix'),'1')]",
+      "apiVersion": "2015-06-15",
```
[the remark this answers is not in the document]
Comment (mgreenegit) [reply 101-automation-configuration/nested/provisionStorage.json]: fixed throughout

File: 101-automation-configuration/nested/provisionStorage.json (hunk 6 above)
Comment (mgreenegit) [reply 101-automation-configuration/nested/provisionStorage.json]: fixed throughout

**Comment by mgreenegit** [comment]
#sign-off @bmoore-msft

**Pull request merged**